packages/security/src/CredentialManagerFactory.ts

```diff
@@ -85,7 +86,7 @@ export class CredentialManagerFactory {
         const displayName = (params.displayName == null) ? params.service : params.displayName;
 
         // If a manager override was not passed, use the default keytar manager
-        const Manager = (params.Manager == null) ? DefaultCredentialManager : params.Manager;
+        const Manager: any = CredentialManagerFactory.determineCredentialManagerType(params.Manager);
```

Comment:
`any` -> `AbstractCredentialManager`  maybe?

Reply:
Sadly I think this one might have to stay as any or we can also add possible types since I saw this comment from @t1m0thyj  from a while back on the old code:

```JS
// After constructing the object, we will ensure that the thing loaded is indeed an
// instance of an abstract credential manager. Since we cannot assume that our internal
// load of a plugin provided a correct object to the function :/
```